module4/projects/mod4Project/inception.md

```diff
@@ -0,0 +1,114 @@
+    * Don’t spend more than 5 minutes on any one feature. Remember, this is just a brainstorming session.
+    * Do not eliminate any ideas during this meeting.
+    * No decisions are made during this meeting
+    * Allow everyone’s voice to be heard and make sure no one is dominating the discussion.
+    * Be creative. Share bad ideas as well as good. Sometimes bad ideas lead to good ones.
+    * Use the guiding questions below if you get stuck.
+* Guiding Questions
+    * What would set you apart from the other groups?
+    * What intimidates you but would be amazing to pull off?
+    * What feature choices would spark interesting discussions during job interviews?
+    * Would you use this product? If not, what is it missing?
+
+### Feature Dump (Brainstorm)
+* **Objectives**
+    * Write out as many potential features as possible
+* **Instructions**
+    * This is an **individual** activity
+    * Each person should write out features on Sticky Notes
+    * Only 1 feature per sticky note
+    * Features should describe user interaction in 3-5 words max.
+    * **If you cannot fit a feature on a single sticky note you are doing it wrong. You are probably providing implementation details. These feature descriptions should describe user interaction and should not be tied to implementing the feature in a specific way**
+    * All potential features of the app should be included (we will define the MVP later in the  process)
+    * Get creative with potential features. There are no wrong answers.
+
+### Feature Sharing/Elimination of Duplicates (Decide)
+
+* **Objectives**
+    * Share what came out of the feature dump
+    * Group related ideas into broad categories (EPICs)
+    * Eliminate duplicate ideas
+* **Instructions**
+  * You will be organizing these thoughts as a group in the project's Miro board
+  * Each person should present their features one at a time, giving a *brief* description of what that feature is, so it is clear for the whole group.
+  * After you have described the feature move the sticky note into a category. If it doesn't fit in an existing category, make one.
+  * As each person takes a turn you will notice that some of the features will be duplicates of previously shared features. If this is the case, simple delete that sticky note. _Don't spend time describing a feature if it is a duplicate._
+  * Come up with a name for each of the categories. These will be referred to as [EPICs](https://www.yodiz.com/blog/what-is-epic-in-agile-methodology-definition-and-template-of-epic/) and will be a way to organize your features in your project management tool.
+
+### Feature Prioritization and MVP Definition (Decide)
+
+* **Objectives**
+    * Define the features that will make up your [**Minimum Viable Product**](https://www.agilealliance.org/glossary/mvp/#q=~(infinite~false~filters~(tags~(~'mvp))~searchTerm~'~sort~false~sortDirection~'asc~page~1))
+* **Instructions (recommended)**
```

Comment:
I would add a comment here that whatever you think you MVP is, you probably want to cut that in half. I think instructors should also help give feedback here on how to scope down the MVP even more.